### ISSUE TITLE
RefreshFirebaseTokens error 

### ISSUE THREAD

**Issue by DPStudioGR** [PrestaShop/PrestaShop#36573]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

One of my Prestahop installations , having version 1.7.8.11 2 days ago stopped working...

In Front office (chrome browser)  shows  HTTP ERROR 500

Backoffice (chrome browser) shows the following error 

-------------------------------------------------------------------------------
"Oops... looks like an unexpected error occurred

Attempted to load trait "RefreshFirebaseTokens" from namespace "PrestaShop\Module\PsAccounts\Account\Session". Did you forget a "use" statement for "PrestaShop\Module\PsAccounts\Account\Session\RefreshFirebaseTokens"?

[Symfony\Component\Debug\Exception\ClassNotFoundException 0]
---------------------------------------------------------------------------------

In server log i get the following errors

 2024-07-18 21:53:09 | Warning | 52.143.70.100 |   | mod_fcgid: stderr: PHP Fatal error: Trait 'PrestaShop\\Module\\PsAccounts\\Account\\Session\\RefreshFirebaseTokens' not found in /var/www/vhosts/buyclick.gr/stampa-stampa.com/modules/ps_accounts/src/Account/Session/ShopSession.php on line 35

I presume this happened due to changes in PS Modules (PrestaShop EventBus /account) cause my other installations at the same time had updates that were done while in this installation when i tried to connect to do the updates i could not connect. 

### Expected behavior

Basically login to back office and frontoffice working on all browsers as before 16/7/24

### Steps to reproduce

1. Go to https://stampa-stampa.com

### PrestaShop version(s) where the bug happened

1.7.8.11

### PHP version(s) where the bug happened

7.4.33  Fastcgi Apache

### Your company or customer's name goes here (if applicable).

Stampa-Stampa.com

**Comment by matks** [comment]
Hello, could you please submit this bug report on https://github.com/PrestaShopCorp/ps_accounts/ ;) ?

**Comment by DPStudioGR** [comment]
Sure...  but how ? i see there is no issue/bug area..

**Comment by florine2623** [comment]
You can do it here : https://github.com/PrestaShopCorp/ps_accounts/issues/new

Thank you !

**Comment by matks** [comment]
Closing as https://github.com/PrestaShopCorp/ps_accounts/issues/421 was opened 😉 thanks
